README.md

```diff
@@ -21,11 +21,11 @@ Actually it contains the following abstractions which has been extracted from bo
 * **Jkube-kit**, which consists of the following:
 
   * **Generator** framework for automatically generating Docker images by examining project information.<br />
-  [![Maven Central](https://img.shields.io/maven-central/v/org.eclipse.jkube/jkube-maven-generator-api.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22org.eclipse.jkube%22%20AND%20a:%22jkube-maven-generator-api%22) [![Javadocs](http://www.javadoc.io/badge/org.eclipse.jkube/jkube-maven-generator-api.svg?color=blue)](http://www.javadoc.io/doc/org.eclipse.jkube/jkube-maven-generator-api)
+  [![Maven Central](https://img.shields.io/maven-central/v/org.eclipse.jkube/jkube-kit-generator-api.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22org.eclipse.jkube%22%20AND%20a:%22jkube-kit-generator-api%22) [![Javadocs](http://www.javadoc.io/badge/org.eclipse.jkube/jkube-kit-generator-api.svg?color=blue)](http://www.javadoc.io/doc/org.eclipse.jkube/jkube-kit-generator-api)
```

Comment:
Shall we update badges after we cut release? WDYT?

Reply:
OK, I'll add an additional commit on top of this one that can be easily reverted after next release.